src/main/java/seedu/address/model/person/Attendance.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Student's Attendance.
+ */
+public class Attendance {
+    private final LocalDateTime date;
+    private final boolean hasAttended;
```

Comment:
Turns out we are going with `private` anyways. To be consistent with #35, can you make the getters for these as well?